cmd/createcluster_internal_test.go

```diff
@@ -330,6 +331,13 @@ func testCreateCluster(t *testing.T, conf clusterConfig, def cluster.Definition,
 			if isAnyVersion(lock.Version, "v1.7.0") {
 				require.NotEmpty(t, val.BuilderRegistration)
 			}
+
+			if conf.SplitKeys {
+				// For SplitKeys mode, builder registration timestamp must be close to Now().
+				// This assumes the test does not execute longer than five minutes.
+				// We just need to make sure the message timestamp is not a genesis time.
+				require.Less(t, nowUTC.Sub(val.BuilderRegistration.Message.Timestamp), 5*time.Minute, "likely a genesis timestamp")
```

Comment:
Hmmm, considering our CI isn't the fastest, I wonder if we'll ever reach a state in which this statement fails?

Reply:
This test is fast, it does not deploy and run a cluster. On a dev box it is executing in ~0.2 sec. Hence, the assumption about 5 minutes to be extremely sufficient..